examples/apollo-tracing.graphql

```diff
@@ -0,0 +1,22 @@
+schema
+  @server(
+    port: 8000
+    graphiql: true
+    hostname: "0.0.0.0"
+    apollo: {api_key: "<api-key>", graph_ref: "tailcall-demo-3@current"}
```

Comment:
we should support mustache templates here. For reference you can check latest merged telemetry pr that allows to use mustache templates for configurations like this